src/architecture/configuration.md

```diff
@@ -58,106 +58,105 @@ List of validators’ public keys as hex strings. Each list element consists of
 
 #### [network]
 
 [Local connection](../advanced/network.md) parameters.
 
 - **max_incoming_connections**  
-  Maximum number of incoming connections.
+  Maximum number of incoming connections
 - **max_outgoing_connections**  
-  Maximum number of outgoing connections.
+  Maximum number of outgoing connections
 - **tcp_nodelay**  
-  Activation of the NODELAY algorithm from the TCP stack (see [RFC2126][rfc2126]).
+  Activation of the NODELAY algorithm from the TCP stack (see [RFC2126][rfc2126])
 - **tcp_reconnect_timeout**  
-  Timeout (ms) for the first attempt to reconnect.
+  Timeout interval (ms) before the first reconnect attempt
 - **tcp_reconnect_timeout_max**  
-  Maximum timeout (ms) for reconnect attempt.
+  Maximum timeout interval (ms) for reconnect attempt
 
 #### [api]
 
 API configuration parameters.
 
 - **enable_blockchain_explorer**  
-  Enable API endpoints for the blockchain explorer on the public API address.
+  Enables API endpoints for the blockchain explorer at the public API address
 - **state_update_timeout**  
-  Timeout (ms) to update info about connected peers.
+  Timeout interval (ms) to update info about connected peers
 - **public_api_address**  
-  Listen address for public API endpoints.
+  Listen address for public API endpoints
 - **private_api_address**  
-  Listen address for private API endpoints.
+  Listen address for private API endpoints
 
 #### [whitelist]
 
 [Network whitelisting](../advanced/network.md#whitelist) parameters.
 
 - **whitelist_enabled**  
-  Enable whitelisting.
+  Enables whitelisting
 - **whitelisted_peers**  
-  List containing consensus public keys of trusted peers.
+  List of consensus public keys for trusted peers
 
 #### [mempool]
 
 Message processing parameters.
 
 - **events_pool_capacity**  
-  Maximum number of events in the [event queue](../advanced/consensus/specification.md#message-processing).
-
+  Maximum number of events in the [event queue](../advanced/consensus/specification.md#message-processing)
 - **tx_pool_capacity**  
-  Maximum number of transactions in the [pool of unconfirmed transactions](../advanced/consensus/specification.md#pool-of-unconfirmed-transactions).
+  Maximum number of transactions in the [pool of unconfirmed transactions](../advanced/consensus/specification.md#pool-of-unconfirmed-transactions)
 
 #### [services_configs]
 
-Service-specific parameters under the keys named according to [`service_name`s](services.md#service-identifiers)
+Service-specific parameters under the keys corresponding to [`service_name`s](services.md#service-identifiers)
 of the blockchain services.
 
 ## Changing Configuration
 
 Global parameters should be changed by using the global variables updater
 service. The service ensures that the configuration updates are synchronized
-among nodes in the network. Global parameters should not be changed
+among nodes in the network. It is not advisable to change global parameters
```

Comment:
It is not advisable to ... -> One should not ...

It is not advice but requirement.